style.css

```diff
@@ -0,0 +1,90 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap");
+
+body {
+  font-family: "Poppins", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 375px;
+  margin: 1px;
+  padding: 1px;
+}
+
+.Logo a {
+  font-size: 20px;
+  color: #6070ff;
+  margin-top: px;
+  padding: 10px;
+}
+
+nav i {
+  position: absolute;
+  top: 5%;
+  right: 5%;
+  color: #6070ff;
+  width: 12px;
+  height: 3px;
+  margin-bottom: 20px;
+}
+
+.Header-main {
+  background-image: url(header-shapes\ mobile.png);
```

Comment:
- [ ] As requested by all previous reviewers, kindly add the background image to this section of your portfolio. I can see you added the URL path to the background image but the image file is missing in your portfolio. Kindly download the image from figma and add it to your design.